src/gurt/shm_dict.c

```diff
@@ -717,21 +718,22 @@ shm_ht_rec_find_insert(d_shm_ht_loc_t shm_ht_loc, const char *key, const int len
 			off_next = rec->next;
 		}
 	}
+	/* add padding space to make sure value is aligned by SHM_MEM_ALIGN */
```

Comment:
why is padding required again here?

Reply:
In case of a robust mutex is stored in the data of hash table record, SHM_MEM_ALIGN (4-byte) aligned is required to make robust mutex working as expected. The padding is enforced to make sure data stored in hash table is 4 bytes aligned.